packages/beacon-node/test/unit/chain/forkChoice/forkChoice.test.ts

```diff
@@ -298,10 +286,7 @@ describe("LodestarForkChoice", function () {
       forkChoice.onBlock(blockZ.message, stateZ, blockDelaySec, blockZ.message.slot, executionStatus);
 
       const head = forkChoice.updateHead();
-      expect(head.blockRoot).to.be.equal(
-        toHexString(ssz.phase0.BeaconBlock.hashTreeRoot(blockY.message)),
-        "blockY should be new head as it's a potential head and has same unrealized justified checkpoints & more attestations"
-      );
+      expect(head.blockRoot).toBe(toHexString(ssz.phase0.BeaconBlock.hashTreeRoot(blockY.message)));
```

Comment:
previous assertion message seems helpful to understand the test case, could consider using `toBeWithMessage` here